eng/pipelines/templates/scripts/docsms-allowlist/python-allowlist.txt

```diff
@@ -0,0 +1,6 @@
+^ci-configs/
+^docs-ref-autogen/
+^docs-ref-mapping/
+^docs-ref-services/
+^metadata/
+^preview/docs-ref-autogen/
```

Comment:
Should also include `^previous/docs-ref-autogen` ... That's the `legacy` moniker but it's built as part of the normal Python docs CI build.

Reply:
Try to be aggressive at the beginning. We don't want to support any legacy update going forward. No legacy helps us understand the frequency of the legacy changes. It doesn't add too much overheads for creating a PR, and this can be changed later on in docs if we know how often it fails.